draft-ounsworth-pq-composite-sigs.txt

```diff
@@ -482,9 +481,9 @@ Internet-Draft             PQ Composite Certs               January 2020
    CompositeSignatureValue ::= SEQUENCE SIZE (1..MAX) OF BIT STRING
 
    Where each BIT STRING within the SEQUENCE is a signature value
-   produced by one of the component keys.  It MUST contain MUST contain
```

Comment:
this should be corrected though, but probably associated with a different issue or larger changes since the linked issue is invalid.

Reply:
I'll add it to larger change.